corehq/apps/userreports/tests/test_columns.py

```diff
@@ -214,20 +215,35 @@ def test_getting_distinct_values(self):
             'apple',
             'banana',
             'blueberry'
-        ])
+        ], [{
+            "type": "expanded",
+            "field": 'my_field',
+            "display": 'my_field',
+            "format": "default",
+        }])
```

Comment:
looks like these got copy/pasted to a bunch of places. could we make it a class attribute to avoid that?